server/data/data-library/src/main/java/org/apache/james/rrt/lib/RecipientRewriteTableManagement.java

```diff
@@ -68,6 +68,18 @@ public void removeAddressMapping(String fromUser, String fromDomain, String toAd
         rrt.removeAddressMapping(source, toAddress);
     }
 
+    @Override
+    public void addUserAliasMapping(String fromUser, String fromDomain, String toAddress) throws RecipientRewriteTableException {
```

Comment:
Should not be needed cf https://github.com/linagora/james-project/pull/3076

Reply:
removed